### DISCUSSION TITLE
chrome flag "--disable-translate" does not work in latest 6.4.1 CLI

### DISCUSSION THREAD

**Discussion by alexd-3d** [GoogleChrome/lighthouse#11860] [Q&A] [answered]
#### Provide the steps to reproduce
1. Run LH on any eng website from non-english speaking country

my cli command:
`lighthouse https://www.thorntons.co.uk --view --perf --quiet --chrome-flags="--disable-translate" --only-categories=performance
`

#### What is the current behavior?
At first: Thanks A LOT for the great tool! )
I'm in Ukraine so testing from lighthouse-cli English sites shows me google translation window on each run. I found in docs that disable-translate should do the work and disable it, but seems like chrome ignores this flag.
I have used it as a chrome-flag and as a lighthouse-cli  parameter (just in case) - none seems to have an effect.

![image](https://user-images.githubusercontent.com/29104318/102699484-cd58dc80-424d-11eb-9d74-43d513354a30.png)

#### What is the expected behavior?
Google translate popup should not be shown on chrome launch.

#### Environment Information
* Affected Channels: CLI
* Lighthouse version: 6.4.1
* Chrome version:  87.0.4280.88 (64 bit)
* Node.js version: 10.14.2
* Operating System: Win10 64bit

Just one more additional issue\question please. If I run the lighthouse-cli with --headless parameter seems like I'm constantly getting ~20-30% higher results than with Chrome opening. Do I miss something? Or there is no cpu\network throttling applied in headless mode? 
Here's what my automated script with computeMedianRun used gives as a result to compare. The difference is almost constant. 
![image](https://user-images.githubusercontent.com/29104318/102699713-94216c00-424f-11eb-8cd1-c108d70ebf8c.png)
I believe that opening Chrome and that google translate window do decrease performance. 
1) Is this expected difference?
2) Should I calibrate --headless runs in this case with higher cpuSlowdownMultiplier value in params to simulate chrome run? I've tried, but seems like "Speed index" and "Time to interactive" is not affected by cpuSlowdownMultiplier. So possibly some other network throttling settings should be also enabled?

Thanks!

**Comment by patrickhulce** [comment, marked as answer]
Thanks for filing @alexd-3d!

`--disable-translate` isn't a Chrome flag and hasn't been [for a number of years](https://github.com/chromium/chromium/commit/4baa4206fac22a91b3c76a429143fc061017f318).

Chrome [recently renamed the `TranslateUI` flag](https://chromium-review.googlesource.com/c/chromium/src/+/2404484) (which Lighthouse disables by default in 6.4.1) to `Translate` which is why this change might be new to you. Use `--chrome-flags="--disable-features=Translate"`.

As for your headless v. non-headless questions. Yes, the performance characteristics can be different especially for graphics operations. There aren't really any Lighthouse options that can be changed to completely account for this. If running in a server environment virtual display options like xvfb can help run headful Chrome for the most accurate results.

[The variability docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) detail many other environmental factors you may run into that cause systemic differences.

**Reply by alexd-3d** [reply]
Many thanks for quick reply!
Did not know it was renamed. Appreciate for answers and help, thanks! )
